pkg/remote/aws/provider.go

```diff
@@ -2,6 +2,7 @@ package aws
 
 import (
 	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/hashicorp/terraform/addrs"
```

Comment:
We cannot use this struct anymore since in the newer version this is unexported (moved under `internal` pkg)

We should have our own struct instead under `pkg/terraform`